pages/docs/extensions/google-calendar.mdx

```diff
@@ -17,19 +17,24 @@ On the `import` tab, you will find numerous fields used to configure the calenda
 
 `Calendars` specify which calendars you would like Roam to read before importing. If you specify more than one, it will read from all of those calendars. You must use the calendar ID provided by Google which you could find in the calendar settings. This will usually be your gmail address, such as `dvargas92495@gmail.com`.
 
-`Include Event Link` hyperlinks the event summary with a link to the google calendar event. This field is ignored if you specify a `Format`.
+`Include Event Link` hyperlinks the event summary with a link to the google calendar event. This field also now applies if you specify a `Format`.
```

Comment:
```suggestion
`Include Event Link` hyperlinks the event summary with a link to the google calendar event. This field also applies if you specify a `Format`.
```

I like the philosophy of Documentation always being in the present tense. The `now` implies some past tense

Reply:
👍